core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -1376,6 +1376,13 @@
 
   public static final String BITSET_PIPE_LINE_DEFAULT = "true";
 
+  /**
+   * It will merge the carbon index files with in the segment to single segment.
```

Comment:
Please mention it is for test purpose only, it should not be set by user

Reply:
ok